Sources/Sampler/metropolis_local_pt.hpp

```diff
@@ -60,9 +58,9 @@ class MetropolisLocalPt : public AbstractSampler {
  public:
   // Constructor with one replica by default
   explicit MetropolisLocalPt(AbstractMachine& psi, int nreplicas = 1)
-      : psi_(psi),
-        hilbert_(psi.GetHilbert()),
-        nv_(hilbert_.Size()),
+      : AbstractSampler(psi.GetHilbert()),
```

Comment:
`AbstractSampler(psi)` would do, then both psi and hilbert are stored in the protected variables